apps/antalmanac/src/components/Calendar/CourseCalendarEvent.tsx

```diff
@@ -294,16 +294,18 @@ const CourseCalendarEvent = (props: CourseCalendarEventProps) => {
         return (
             <Paper className={classes.customEventContainer} ref={paperRef}>
                 <div className={classes.title}>{title}</div>
-                <div className={classes.table}>
-                    Location: &nbsp;&nbsp;
-                    <Link
-                        className={classes.clickableLocation}
-                        to={`/map?location=${building ?? 0}`}
-                        onClick={focusMap}
-                    >
-                        {building ? buildingCatalogue[+building].name : ''}
-                    </Link>
-                </div>
+                {building && (
+                    <div className={classes.table}>
+                        Location:&nbsp;
+                        <Link
+                            className={classes.clickableLocation}
+                            to={`/map?location=${building ?? 0}`}
+                            onClick={focusMap}
+                        >
+                            {buildingCatalogue[+building].name}
+                        </Link>
+                    </div>
+                )}
```

Comment:
I took the liberty of adjusting this piece of code, since it would often display `Location: `, if there was no selected building.